<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>LoginForm</class>
    <widget class="QDialog" name="LoginForm">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>700</width>
                <height>500</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>700</width>
                <height>500</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>700</width>
                <height>500</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <property name="styleSheet">
            <string notr="true" />
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item alignment="Qt::AlignHCenter">
                <widget class="QLabel" name="label">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>30</pointsize>
                        </font>
                    </property>
                    <property name="text">
                        <string>Login</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <property name="spacing">
                        <number>30</number>
                    </property>
                    <property name="leftMargin">
                        <number>20</number>
                    </property>
                    <property name="topMargin">
                        <number>20</number>
                    </property>
                    <property name="rightMargin">
                        <number>20</number>
                    </property>
                    <property name="bottomMargin">
                        <number>20</number>
                    </property>
                    <item>
                        <widget class="QLabel" name="label_5">
                            <property name="font">
                                <font>
                                    <pointsize>20</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Username</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="lineEdit_username">
                            <property name="maximumSize">
                                <size>
                                    <width>400</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>20</pointsize>
                                </font>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <property name="spacing">
                        <number>30</number>
                    </property>
                    <property name="leftMargin">
                        <number>20</number>
                    </property>
                    <property name="topMargin">
                        <number>20</number>
                    </property>
                    <property name="rightMargin">
                        <number>20</number>
                    </property>
                    <property name="bottomMargin">
                        <number>20</number>
                    </property>
                    <item>
                        <widget class="QLabel" name="label_6">
                            <property name="font">
                                <font>
                                    <pointsize>20</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Password</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="lineEdit_passowrd">
                            <property name="maximumSize">
                                <size>
                                    <width>400</width>
                                    <height>46</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>20</pointsize>
                                </font>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QPushButton" name="pButton_newUser">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>200</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>20</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>New User</string>
                            </property>
                            <property name="autoDefault">
                                <bool>false</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="pButton_login">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>200</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>20</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Login</string>
                            </property>
                            <property name="autoDefault">
                                <bool>false</bool>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="pButton_specialUser">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="minimumSize">
                        <size>
                            <width>200</width>
                            <height>0</height>
                        </size>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>200</width>
                            <height>16777215</height>
                        </size>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>20</pointsize>
                        </font>
                    </property>
                    <property name="text">
                        <string>Special User</string>
                    </property>
                    <property name="autoDefault">
                        <bool>false</bool>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>
